autoflake.py

```diff
@@ -625,13 +632,20 @@ def fix_file(filename, args, standard_out):
 
     original_source = source
 
+    if args.ignore_init_module_imports and filename.endswith('__init__.py'):
```

Comment:
For the part to the right of `and`, can you use [`os.path.basename()`](https://docs.python.org/2/library/os.path.html#os.path.basename)? Right now it looks like this might match against `hello__init__.py`.

Reply:
My apologies for the long delay, I just saw this comment today after stumbling upon this PR again. Changes have been made.